tests/devices/test_lightning_qubit.py

```diff
@@ -97,6 +89,7 @@ def circuit():
     assert np.allclose(circ0(), circ1(), rtol=0.01)
 
 
+@pytest.mark.skip(reason="New lightning.qubit needs to be updated to work with dtypes")
```

Comment:
TODO: Remove real dtype test, stop skipping class after c_dtype issue with lightning is fixed.